backend/coreapp/views.py

```diff
@@ -13,18 +13,14 @@
 from .models import Profile, Asm, Scratch
 from coreapp.models import gen_scratch_id
 
+
 def get_db_asm(request_asm) -> Asm:
     h = hashlib.sha256(request_asm.encode()).hexdigest()
+    asm, _ = Asm.objects.get_or_create(hash=h, defaults={
```

Comment:
Should probably handle the error

Reply:
not an error, just a boolean saying whether it already exists